modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/meetings/portlet/MeetingsPortlet.java

```diff
@@ -488,15 +496,7 @@ public void updatePowwowMeeting(
 					serviceContext);
 			}
 
-			String newRecurrenceHash = _getRecurrenceHash(calendarBooking);
-
-			boolean recurrenceChanged = !oldRecurrenceHash.equals(newRecurrenceHash);
-
 			if (recurrenceChanged) {
-				if (powwowMeetingId > 0) {
-					_deletePowwowMeetingOccurrence(powwowMeetingId);
-				}
-
 				if (calendarBooking.isRecurring()) {
```

Comment:
should move the condition `calendarBooking.isRecurring()` to the outer `if`, so that we can reduce this nested `if`